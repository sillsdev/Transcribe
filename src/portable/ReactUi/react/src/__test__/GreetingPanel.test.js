/* jshint esversion: 6 */
import React from 'react';
import expect from 'expect';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import sinon from 'sinon';
import GreetingPanel from '../components/GreetingPanel';


configure({ adapter: new Adapter() })

// Snapshot for GreetingPanel
describe('>>>Control: GreetingPanel --- Snapshot',()=>{
    const minProps = {
        strings: { congratulations: "Congratulations", youhavereached: "You have reached", haveaniceday: "Have a nice day", inboxzero: "zero inbox"}
    };

    it('+++capturing Snapshot of GreetingPanel', () => {
        const renderedValue =  renderer.create(<GreetingPanel {...minProps}/>).toJSON()
        expect(renderedValue).toMatchSnapshot(1);
    });
});

