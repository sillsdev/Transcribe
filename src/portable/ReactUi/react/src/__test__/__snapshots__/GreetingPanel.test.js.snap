// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1 1`] = `
<div
  className="GreetingPanel"
>
  <div
    className="Message"
  >
    <img
      alt="Wow!!"
      height="50%"
      src="/assets/Smile.svg"
      width="50%"
    />
    <h1>
      Congratulations
      !
      <br />
      You have reached
       
      <br />
      zero inbox
      . 
    </h1>
    <div>
      <h1>
        Have a nice day
      </h1>
    </div>
  </div>
</div>
`;
